Source/WebCore/loader/PolicyChecker.cpp

```diff
@@ -311,7 +311,8 @@ void PolicyChecker::checkNavigationPolicy(ResourceRequest&& request, const Resou
 
     if (isInitialEmptyDocumentLoad) {
         // We ignore the response from the client for initial empty document loads and proceed with the load synchronously.
-        frameLoader->client().dispatchDecidePolicyForNavigationAction(action, request, redirectResponse, formState.get(), clientRedirectSourceForHistory, navigationID, hitTestResult(action), hasOpener, isPerformingHTTPFallback, sandboxFlags, policyDecisionMode, [](PolicyAction) { });
+        if (!frame->settings().siteIsolationEnabled())
```

Comment:
I don't think we should do this.  This will also cause an unnecessary bincompat issue when we turn it on.  We should fix the incorrect process selection instead.